src/components/Layout/LayoutHeader.tsx

```diff
@@ -148,7 +175,50 @@ export const LayoutHeader = ({
             )}
           </div>
         </View>
+        <div className="">
```

Comment:
Nit - Doesn't look like this div is useful when the ToC is not showing, if it doesn't change the page layout any can we move it into the conditional render so that there isn't just an empty div when the ToC isn't showing?